packages/tao/src/commands/ngcli-adapter.ts

```diff
@@ -858,6 +858,12 @@ export function wrapAngularDevkitSchematic(
 
       if (event.kind === 'error') {
       } else if (event.kind === 'update') {
+        if (
+          r.eventPath === '/angular.json' ||
```

Comment:
Is the `/` significant here? Why does `angular.json` have it, but `workspace.json` not?

Reply:
Good catch, neither should have 👍